packages/client/src/context.ts

```diff
@@ -133,6 +133,26 @@ export async function createClientContext(
           triggerEndpoint: TRIGGER_CLIENT_ENDPOINTS.mainnet,
         };
       }
+      if (opts === 'blastTestnet') {
```

Comment:
Do you have bandwidth to clean this up a bit? This fn is quite large and there's a decent bit of duplication

Reply:
not immediately, but can add a backlog item to address.